### ISSUE TITLE
Update PHPMailer req

### ISSUE THREAD

**Issue by KarelWintersky** [PHPAuth/PHPAuth#281]
```
→ composer require phpauth/phpauth
Using version ^1.1 for phpauth/phpauth
./composer.json has been updated
Loading composer repositories with package information
Updating dependencies (including require-dev)
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - phpauth/phpauth v1.1 requires phpmailer/phpmailer ^5.2 -> satisfiable by phpmailer/phpmailer[v5.2.10, v5.2.11, v5.2.12, v5.2.13, v5.2.14, v5.2.15, v5.2.16, v5.2.17, v5.2.18, v5.2.19, v5.2.2, v5.2.20, v5.2.21, v5.2.22, v5.2.23, v5.2.24, v5.2.25, v5.2.26, v5.2.4, v5.2.5, v5.2.6, v5.2.7, v5.2.8, v5.2.9] but these conflict with your requirements or minimum-stability.
    - phpauth/phpauth 1.1.1 requires phpmailer/phpmailer ^5.2 -> satisfiable by phpmailer/phpmailer[v5.2.10, v5.2.11, v5.2.12, v5.2.13, v5.2.14, v5.2.15, v5.2.16, v5.2.17, v5.2.18, v5.2.19, v5.2.2, v5.2.20, v5.2.21, v5.2.22, v5.2.23, v5.2.24, v5.2.25, v5.2.26, v5.2.4, v5.2.5, v5.2.6, v5.2.7, v5.2.8, v5.2.9] but these conflict with your requirements or minimum-stability.
    - Installation request for phpauth/phpauth ^1.1 -> satisfiable by phpauth/phpauth[1.1.1, v1.1].

```

Latest version of PHPMailer is `v6.0.3`

**Comment by phrenopolis** [comment]
Speaking of PHPMailer versions...
I've been working with PHPAuth for a couple of weeks and have tracked my recent inability to get email verifications to work to an apparent PHPMailer version incompatibility. To wit:
The Auth.php file imports the PHPMailer namespace (use PHPMailer\PHPMailer\PHPMailer;). The version of PHPMailer that composer has chosen to install is 5.2.26, which does not define a namespace, apparently they started doing that in later versions. When Auth tries to create a PHPMailer object (in the addRequest method), it generates a fatal error because it tries to find PHPMailer\PHPMailer\PHPMailer and can't.
So, I would also be in favor of changing the required version of PHPMailer, to one that defines the namespace Auth is expecting.

**Comment by phrenopolis** [comment]
Composer did not balk for me like it did for Karel, but it did install PHPMailer version 5.2.26.
The trouble with that is that Auth.php treats PHPMailer as having a defined namespace - which it apparently doesn't in version 5.2.26.
This causes a fatal error when the addRequest method tries to create a PHPMailer object - it can't find the class because it's looking in the wrong place.
...Which suggests that the dependency for PHPMailer should indeed be changed to a more modern version...

**Comment by phrenopolis** [comment]
(Or, of course, a backslash could be added to the single line in Auth.php where PHPMailer is referenced directly, causing it to be found in the global scope - that's what I did and it all works great now. But that would be less "update PHPMailer req" and more "change code to match current PHPMailer req".)

**Comment by louis123562** [comment]
Where did you add the backslash ?

use PHPMailer\PHPMailer\PHPMailer; //here ??

**Comment by phrenopolis** [comment]
@louis123562 No, down where it's called to create an instance. The line was

  $mail = new PHPMailer;

and I changed it to

  $mail = new \PHPMailer;

which tells it to go look in the global namespace for the class. PHPMailer version 5 defines the class outside of any namespace, so global is where it can be found.

**Comment by louis123562** [comment]
Ah thank you. That works!

**Comment by KarelWintersky** [comment]
fixed in PR #294
